mnemonic-computing-services/mnemonic-utilities-service/src/main/java/org/apache/mnemonic/service/computing/internal/PrintServiceImpl.java

```
@@ -50,6 +50,11 @@ public String getServiceId() {
     return ret;
   }
 
+  @Override
+  public long[] perform(String mode, ValueInfo[] valinfos, long dcHandler, long dcSize) {
+    return null;
```

It is better to throw UnsupportedOperationException please,